src/routes/user.js

```diff
@@ -1,19 +1,97 @@
+/**
+ * @swagger
+ * tags:
+ *    name: User
+ *    description: API to manage users.
+ */
+
 import express from 'express';
 import User from '../controllers/user';
 
 const router = express.Router();
 
-// TODO: Add Swagger Docs
-
-// add a new user
+/**
+ * @swagger
+ * path:
+ * /users/:
+ *    post:
+ *      description: Creates a new user
+ *      summary: Creates a user
+ *      tags:
+ *        - User
+ *
+ *      requestBody:
+ *        description:
+ *        required: true
+ *
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
+ *                username:
+ *                  type: string
+ *                  example: jbrown
+ *                password:
+ *                  type: string
+ *                  example: password12345
+ *                firstName:
+ *                  type: string
+ *                  example: John
+ *                lastName:
+ *                  type: string
+ *                  example: Brown
+ *
+ *      produces:
+ *        - application/json
+ *      responses:
+ *        200:
+ *          description: OK
+ *        404:
+ *          description: Username/password combination is incorrect
```

Comment:
You can't have an incorrect combination on user creation. An error would occur if the username is already taken as they are unique.

Reply:
Good catch. I'll fix it as well.